Formula/kertish-dfs.rb

```diff
@@ -0,0 +1,30 @@
+class KertishDfs < Formula
+  desc "Kertish FileSystem and Cluster Administration CLI"
+  homepage "https://github.com/freakmaxi/kertish-dfs"
+  url "https://github.com/freakmaxi/kertish-dfs/archive/v21.2.0066.tar.gz"
+  sha256 "1ff300a803a0d3bd068e1492319a771912cccd4ba3a72098b6d05678813af966"
+  license "GPL-3.0-only"
+  head "https://github.com/freakmaxi/kertish-dfs.git"
+
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "mod", "download"
+    Dir.chdir("fs-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtfs" }
+    Dir.chdir("admin-tool") { system "go", "build", "-trimpath", "-ldflags", "-X main.version=#{version}", "-o", "#{bin}/krtadm" }
+  end
+
+  def post_install
+    puts "", "", "If you need Kertish Node Executables for v#{version}, please visit https://github.com/freakmaxi/kertish-dfs/releases/tag/v21.2.0066", "", ""
+  end
+
+  test do
+    system "#{bin}/krtfs", "-v"
+    system "#{bin}/krtadm", "-v"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Actually, I was unable to find a way to test it other than getting the version number. The reason is, this formula is only for CLI tools. CLI tools need the DFS cluster setup to work. This means, one manager node, one head node, and at least two data nodes and they should be running to test the CLI commands. Otherwise, commands will not provide any positive output. I'm open to having some ideas...

Reply:
How about
>If the software cannot function without credentials or requires a virtual machine, docker instance, etc. to run, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.

https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula